Jenkinsfile

```diff
@@ -234,6 +235,14 @@ node (node_label) {
 			}
 		}
 
+		if (webfuse_install == "TRUE") {
```

Comment:
Can we not use POST_CONFIG flag within the install job to actually kick webfuse as a part of the job @mukrishn ?

Reply:
Yes we can, you mean existing `OPENSHIFT_POST_CONFIG` ? if its not meant for any other job I can use this one.

Reply:
IYes, I'm thinking we probably don't even need a new job for webfuse install and can consume `OPENSHIFT_POST_CONFIG` in the deployment job itself and configure the cluster...what are your thoughts?

Reply:
But I still thought to read it from pipeline and keep `webfuse` a separate job with dedicated property file.

Reply:
Alright, that should be OK. When is your current job run, it seems like it is run after all stages?

Reply:
Also, the job can be only run as part of pipeline, correct?

Reply:
Never mind, I think all other stages are disabled for baremetal, so you are installing this after stage 2?

Reply:
Right, so as agreed lets move `webfuse` job next to `deployment`. 
Flow would be Deploy > Webfuse > Benchmark > Scaleup > Benchmark